### DISCUSSION TITLE
Error creating the dinamic configuration

### DISCUSSION THREAD

**Discussion by Dragnell87** [nginx/ansible-role-nginx-config#375] [General]
Hello i am traing to reuse a playbook some one left behind, but it fail when creating the configuration.

I am unable to pickup what is the actual problem, i attatch the ansible debug output. If you found time, can you point me in the right direction? Thanks.

[ansible-debug.txt](https://github.com/nginxinc/ansible-role-nginx-config/files/12784842/ansible-debug.txt)

**Comment by alessfg** [comment]
Could you also paste the playbook you are running? It's quite hard to debug as is -- it seems as if there's a missing required parameter that's causing things to crash.
